tests/Feature/Account/CommandHandler/UpgradeModuleHandlerTest.php

```diff
@@ -25,6 +25,8 @@ public function set_up()
     {
         parent::set_up();
 
+        $this->markTestSkipped('Implement cookie management with cURL');
```

Comment:
:eyes:

Reply:
It's just a header, right ? :trollface:, maybe e2e tests can deal with it and we can remove that test here

Reply:
yes or create a ticket on jira to add test later